README.md

```diff
@@ -1,581 +0,0 @@
-
-  ```js
-  // BAD
-  if (true === foo) { ... }
-  if ('test' === bar) { ... }
-
-  // GOOD
-  if (foo === true) { ... }
-  if (bar === 'test') { ... }
-  ```
-
-  - Variable names should be camelCased except properties*[camelcase]
-
-  ```js
-  // BAD
-  const not_camelcased = 'NOOOO!'
-  import { no_t_camel } from 'bar'
-
-  // GOOD
-  const camelCase = 'YEY'
-  import { camelCased } from 'foo'
-  obj = {
-    test_me: 5
-  }
-  ```
-
-  - There should be no commas on the last element of an array or object[comma-dangle]
-
-  ```js
-  // BAD
-  const badArray = [
-    foo,
-    bar,
-  ]
-  const badObj = {
-    foo: 'test',
-    bar: 'me',
-  }
-
-  // GOOD
-  const goodArr = [
-    foo,
-    bar
-  ]
-  const goodObj = {
-    foo: 'test',
-    me: 'bar'
-  }
-  ```
-
-## Conditional Statement
-
-  - No constant expressions in conditions[no-constant-condition]
-
-  ```js
-  // BAD
-  if (false) {
-    doSomething()
-  }
-
-  while (true) {
-    doSomething()
-  }
-  ```
-
-  - No empty block statements[no-empty]
-
-  ```js
-  // BAD
-  if (foo) {
-  }
-
-  While () {
-  }
-
-  // GOOD
-  if (wat) {
-    // Comment here
-  }
-  ```
-
-  - No case statement fallthrough[no-fallthrough]
-
-  ```js
-  // BAD
-  switch (f) {
-    case 1:
-      doSomething()
-    case 2:
-      doAnything()
-  }
-
-  // GOOD
-  switch (f) {
-    case 1:
-      doSomething()
-      break;
-    case 2:
-      return doAnything()
-    case 3:
-      ...
-  }
-
-  switch (f) {
-    case 1:
-    case 2:
-      return doForBoth()
-  }
-  ```
-
-## Whitespace
-
-  - There should be no spacing on array brackets except for deconstructing array[array-bracket-spacing]
-
-  ```js
-  // BAD
-  const a = [ 'test' ]
-  const b = [ 'foo', 'arf']
-  var [x,y,x] = a
-
-  // GOOD
-  const a = ['boo']
-  const c = ['foo', 'bar']
-  var [ x, y, z ] = a
-  ```
-
-  - Spacing on commas should be after the commas[comma-spacing]
-
-  ```js
-  // BAD
-  var arr = [ 1 ,2 ]
-  var c = [ 1 , 2 ]
-
-  // GOOD
-  var b = [ 2, 3 ]
-  ```
-
-  - Indention on blocks should be 2 spaces[indent]
-
-  ```js
-  // BAD
-  function fn () {
-        wee()
-  }
-
-  // GOOD
-  function test () {
-    yey()
-  }
-  ```
-
-  - Spacing on object keys should be after the colon and not before[key-spacing]
-
-  ```js
-  // BAD
-  const obj = { foo :5 }
-  const a = { poke : 'er' }
-
-  // GOOD
-  const obj = { foo: 'bar' }
-  ```
-
-  - Spacing on keywords should be before and after[keyword-spacing]
-
-  ```js
-  // BAD
-  if(test) { ... }
-  function fn() { ... }
-  if (true){ ... }
-
-  // GOOD
-  if (test) { ... }
-  function fn () { ... }
-  ```
-
-  - No multiple empty lines[no-multiple-empty-lines]
-
-  ```js
-  // BAD (MAXIMUM 2)
-  const a = 5
-
-
-
-  cosnt b = 3
-
-  // OKAY
-  const a = 5
-
-
-  const b = 3
-
-  // GOOD
-  const a = 5
-
-  const b = 4
-  ```
-
-  - There should be a space after open curly brace and before close curly brace for objects and destructor[object-curly-spacing]
```

Comment:
I can't do it. It's too hard and also it wants me to write:
`this.state = { reasonList: []}` whhyyyyy?